feat: UTC-320: Precompute task agreement scores

hakan458 opened the pull request:
Introduces new field `precomputed_agreement` to `Task` model
Excluded from serializers as it is meant for internal use

Commit:
feat: UTC-320: Precompute task agreement scores

Commit:
change field name

Commit:
exclude new field properly

Commit:
Merge branch 'develop' of github.com:heartexlabs/label-studio into fb-utc-320

Commit:
exclude new field from exports

Commit:
exclude new field from task serializers

Commit:
remove index

Commit:
Merge branch 'develop' of github.com:heartexlabs/label-studio into fb-utc-320

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/18410899886)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18410899886

mcanu committed:
Merge branch 'develop' into 'fb-utc-320'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18410899886

Commit:
Merge branch 'develop' of github.com:heartexlabs/label-studio into fb-utc-320

Commit:
Merge branch 'develop' of github.com:heartexlabs/label-studio into fb-utc-320
